.github/workflows/actions/create-ios-sig-creds/action.yml

```diff
@@ -27,8 +27,9 @@ runs:
       shell: bash
       env:
         CERTIFICATE: ${{ inputs.certificate }}
+        certificate_password: ${{ inputs.certificate_password }}
```

Comment:
nit: I would suggest we use screaming snake case for our env variables. I know these values are slightly different, but for consistency I think they should be the same casing as the other values.

ie: CERTIFICATE_PASSWORD